Cargo.toml

```diff
@@ -218,3 +219,7 @@ zksync_crypto_primitives = { path = "core/lib/crypto_primitives" }
 zksync_node_framework = { path = "core/node/node_framework" }
 zksync_eth_watch = { path = "core/node/eth_watch" }
 zksync_shared_metrics = { path = "core/node/shared_metrics" }
+
+# FIXME: remove after consensus crates are updated
+[replace]
```

Comment:
The [`patch` section](https://doc.rust-lang.org/stable/cargo/reference/overriding-dependencies.html#the-patch-section) is a recommended way to do such overrides, but it doesn't seem to work in this case because it disallows the override to point at the same git repo as the original dep (not sure why).